packages/desktop-client/src/components/schedules/ScheduleDetails.tsx

```
@@ -115,7 +208,9 @@ export function ScheduleDetails({ id, transaction }) {
             throw new Error('Unknown field: ' + action.field);
           }
 
-          const fields = { [action.field]: action.value };
+          const fields: { [key: string]: typeof action.value | undefined } = {
+            [action.field]: action.value,
+          };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)